pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>35</version>
+        <version>45</version>
```

Comment:
why 45 and not 46?

Reply:
Because I did not realise we had 46 :-) I'll create a follow-up PR.